### PR TITLE
Add net-certmanager to mega config

### PR DESCRIPTION
**Pull request by ZhiminXiang**
Follow the PR https://github.com/knative/test-infra/pull/1840/files# to configure net-certmanager

### CODE REVIEW

File: config/prow/config_knative.yaml [hunk 1]
```diff
@@ -308,6 +308,16 @@ presubmits:
     - unit-tests: true
     - integration-tests: true
     - go-coverage: true
+
```
Comment (mattmoor) [inline config/prow/config_knative.yaml]: Remove trailing whitespace:
```suggestion

```

File: config/prow/config_knative.yaml [hunk 2]
```diff
@@ -915,6 +925,20 @@ periodics:
       go113: true
       env-vars:
       - ORG_NAME=google
+
```
Comment (mattmoor) [inline config/prow/config_knative.yaml]: Remove trailing whitespace:
```suggestion

```

File: config/prow/config_knative.yaml [hunk 3]
```diff
@@ -915,6 +924,19 @@ periodics:
       go113: true
       env-vars:
       - ORG_NAME=google
+  knative/net-certmanager:
```
Comment (mattmoor) [inline config/prow/config_knative.yaml]: Remove trailing whitespace:
```suggestion

```

File: config/prow/config_knative.yaml [hunk 4]
```diff
@@ -915,6 +924,18 @@ periodics:
       go113: true
       env-vars:
       - ORG_NAME=google
+    - continuous: true
```
Comment (mattmoor) [inline config/prow/config_knative.yaml]: I think you are missing a heading here.

File: config/prow/config_knative.yaml [hunk 5]
```diff
@@ -308,6 +308,15 @@ presubmits:
     - unit-tests: true
     - integration-tests: true
     - go-coverage: true
+  knative/net-certmanager:
```
Comment (mattmoor) [inline config/prow/config_knative.yaml]: nit: space between

Comment (ZhiminXiang) [reply config/prow/config_knative.yaml]: Done

File: config/prow/config_knative.yaml (hunk 2 above)
Comment (mattmoor) [inline config/prow/config_knative.yaml]: Remove trailing whitespace:
```suggestion

```

File: config/prow/config_knative.yaml (hunk 4 above)
Comment (ZhiminXiang) [reply config/prow/config_knative.yaml]: Good catch. I should trust the suggested commit...

**Comment by chaodaiG** [comment]
Please run ./hack/update-codegen.sh

**Comment by chaodaiG** [comment]
This change should auto deploy right? Any way it can comment back on this PR once it's done?

**Comment by mattmoor** [comment]
I believe it does

**Pull request merged**